lib/useUserName.js

```diff
@@ -0,0 +1,42 @@
+import useSWR from 'swr'
+import { getToken } from './userAuth'
+
+export const USER_URL = `${process.env.NEXT_PUBLIC_API_URL}/user`
+
+const useUserName = userId => {
+  // console.log(user_id)
```

Comment:
Remove debugging comment.
I see there are multiple cases in other files too, please check them.